frontend/package.json

```diff
@@ -14,7 +14,7 @@
     "dev": "vite",
     "start": "vite",
     "start:prod": "vite build && vite preview",
-    "start:sandbox": "UNLEASH_API=https://sandbox.getunleash.io/ospro yarn run start",
+    "start:sandbox": "UNLEASH_API=https://sandbox.getunleash.io/ VITE_TEST_REDIRECT=true UNLEASH_BASE_PATH=/pro/ yarn run start",
```

Comment:
Shouldn't this include /pro in UNLEASH_API? or does the proxy automatically append UNLEASH_BASE_PATH to UNLEASH_API?

Reply:
I just copied what we have for enterprise sandbox